hash.c

```diff
@@ -6964,6 +6965,67 @@ static const rb_data_type_t env_data_type = {
  *  - For either method #fetch or method #fetch_values,
  *    you can specify a per-key default via a block.
  *
+ *  === \Hash Comparison
```

Comment:
I'm not sure what the purpose of this section is. This should just be in the documentation of `#==`, `#>`, `#<`, etc.

Reply:
Closing.